components/facets/facet-section.js

```diff
@@ -49,60 +48,23 @@ ClearAll.propTypes = {
  * facet-group button, display the modal containing the facets for that group.
```

Comment:
need to change the documentation here since we don't use facet-group anymore?

Reply:
Yes.